.travis.yml

```diff
@@ -15,6 +15,10 @@ before_install:
   - export CXX="g++-4.9"
   
 install:
+    # Use the following commands one after the other to install everything on Arch using Pacaur:
+    # $ sudo pacman -S git qt5-base qt5-multimedia go python2 mesa gcc && 
+    # $ pacaur -y lcov && 
+    # $ gem install coveralls-lcov
```

Comment:
1. That should not be in the `.travis.yml`, it should be in the `README.md` as well.
2. `lcov` is needed only for CIs (and maybe for some advanced stuff), so it should not be mentioned.

So overall it should be like:
Use the following commands one after the other to install everything on Arch using Pacaur:
`sudo pacman -S git qt5-base qt5-multimedia go python2 mesa gc`
somewhere in the `README.md`.